api/apps/api/src/modules/access-control/projects-acl/project-acl.controller.ts

```diff
@@ -0,0 +1,96 @@
+import { apiGlobalPrefixes } from '@marxan-api/api.config';
+import {
+  Controller,
+  Get,
+  Req,
+  Param,
+  ParseUUIDPipe,
+  UseGuards,
+  Delete,
+  Patch,
+  Body,
+  HttpCode,
+  ForbiddenException,
+} from '@nestjs/common';
+import { JwtAuthGuard } from '@marxan-api/guards/jwt-auth.guard';
+import { ProjectAclService } from './project-acl.service';
+import { RequestWithAuthenticatedUser } from '@marxan-api/app.controller';
+import {
+  ApiBearerAuth,
+  ApiOkResponse,
+  ApiOperation,
+  ApiResponse,
+  ApiTags,
+} from '@nestjs/swagger';
+import { UserRoleInProjectDto } from './dto/user-role-project.dto';
+
+@UseGuards(JwtAuthGuard)
+@ApiBearerAuth()
+@ApiTags('Projects-Users Roles')
+@Controller(`${apiGlobalPrefixes.v1}/roles/projects`)
+export class ProjectAclController {
+  constructor(private readonly projectAclService: ProjectAclService) {}
+
+  @Get(':projectId/users')
+  @ApiOperation({ summary: 'Get all users with roles in project' })
+  @ApiOkResponse({
+    status: 200,
+    description: 'User roles found',
+    type: [UserRoleInProjectDto],
```

Comment:
one-element array of `UserRoleInProjectDto` type?

Reply:
I did not know exactly how to indicate to Swagger that this was an array and I wrongly assumed this was the correct way. Thanks for pointing it out, I think it is solved now.